google-cloud-tasks/lib/google/cloud/tasks/v2beta2/cloudtasks_pb.rb

```diff
@@ -50,7 +50,6 @@
   add_message "google.cloud.tasks.v2beta2.ListTasksRequest" do
     optional :parent, :string, 1
     optional :response_view, :enum, 2, "google.cloud.tasks.v2beta2.Task.View"
-    optional :order_by, :string, 3
```

Comment:
Wild. Is there consensus on how we handle removals in the protobuf for these generated gems? For hand-coded gems we considered the generated layer to not be part of the public API as far as breaking changes goes.

Reply:
I was under the assumption that since this is a prerelease API version (v2beta2), backward-incompatible changes are possible and we'll just have to roll with it. I guess it might be possible to partial-gapic a tombstone field into the class so that clients at least won't NameError, but it seems like a lot of effort.